src/amo/components/SectionLinks/styles.scss

```diff
@@ -67,7 +67,7 @@
   .DropdownMenu-button {
     display: block;
     margin-top: 2px;
-    padding: 0;
+    padding: 0 0 2px;
```

Comment:
This prevents any issues when moving the mouse from the button to the menu–if this isn't here it might cause a `mouseLeave` event to be fired and the menu will disappear.